nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -21,13 +21,14 @@ bindings:
   #   single host: jsonapi_host=host1
   #   multiple hosts: jsonapi_host=host1,host2,host3
   #   multiple weighted hosts: jsonapi_host=host1:3,host2:7
-  weighted_hosts: WeightedStrings('<<jsonapi_host:jsonapi>>')
+  weighted_hosts: WeightedStrings('<<jsonapi_host:stargate>>')
```

Comment:
I am still like why don't we have `localhost` as default? In addition, this change is also only available in this file, not the others.